mozci/data/sources/treeherder/__init__.py

```diff
@@ -152,11 +158,9 @@ def run_push_tasks(self, branch, rev):
 
         return [
             {
-                "id": task_data["id"],
-                "label": task_data["label"],
-                "result": task_data["result"],
-                "duration": task_data["duration"],
-                "tags": task_data["tags"],
+                key: task_data[key]
+                for key in keys
+                if not (key == "result" and task_data[key] == "unknown")
```

Comment:
This should cause an exception if there are classifications, as the returned objects would contain some keys that the contract does not expect

Reply:
Oh forget it, I see you have a list of keys and you are not doing task_data.keys ;)